src/cryptoadvance/specter/templates/base.jinja

```diff
@@ -59,6 +59,7 @@
 	{% endif %}
 	{% include "includes/message-box.html" %}
 	{% include "includes/tool-tip.html" %}
+	{% include "includes/text-explainer.html" %}
```

Comment:
I'd rather not include it in the base template but only in those jinja templates where we need it, especially since we only use it twice so far.

Reply:
Ahh agree, I initially thought that text-explainer can be used at multiple places so did like that. I’ll update its import to specific files since sample space is small right now.